Rules/CommonScripts/MapVotesCommon.as

```diff
@@ -571,22 +571,23 @@ void CreateGenTexture(string shortname)
 
 void CreateMapTexture(string shortname, string filename)
 {
+	print("creating minimap" + ", " + shortname + ", " + filename);
```

Comment:
accidental debug print? otherwise merge the first comma into "creating minimap"

Reply:
fixed